pulsar-transaction/common/src/main/java/org/apache/pulsar/transaction/common/TxnID.java

```diff
@@ -0,0 +1,46 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pulsar.transaction.common;
```

Comment:
Are clients expected to use this package? Or is it server side only?

Reply:
all the classes in transaction are meant to be internal classes. so they are not end-user facing classes. any end-user facing classes should be added to `pulsar-client-api` or `pulsar-common`.

Reply:
If it's internal, it might be worth to add "impl" in the pkg name. That will help disambiguate the scope of the class/interface.

Reply:
moved them to `impl`